core/src/test/kotlin/fr/sncf/osrd/sim_infra_adapter/PathPropertiesTests.kt

```diff
@@ -134,7 +134,8 @@ class PathPropertiesTests {
                         RJSOperationalPointSncfExtension(0, "BV", "B", "0", "TRI"),
                         null,
                     ),
-                    null
+                    null,
+                    false
```

Comment:
As the CH is "BV" above:
```suggestion
                    true
```